litesdcard/phy.py

```diff
@@ -62,14 +62,14 @@ def __init__(self):
         # SDCard CE Generation.
         clk_d = Signal()
         self.sync += clk_d.eq(clk)
-        self.sync += self.ce.eq(clk & ~clk_d)
+        self.comb += self.ce.eq(clk & ~clk_d)
 
         # Ensure we don't get short pulses on the SDCard Clk.
         ce_delayed = Signal()
         ce_latched = Signal()
         self.sync += If(clk_d, ce_delayed.eq(self.clk_en))
         self.comb += If(clk_d, ce_latched.eq(self.clk_en)).Else(ce_latched.eq(ce_delayed))
-        self.sync += self.clk.eq(~clk & ce_latched)
+        self.comb += self.clk.eq(~clk & ce_latched)
```

Comment:
@enjoy-digital : I've understood and agree with all the *rest* of the changes in commits 1a3d474 and 15bc2db, but not sure why `self.ce` and `self.clk` were switched to `sync` from the original `comb`. It is the reason sdcard breaks at lower (50MHz) system clock frequencies.

To be honest, though, I also don't get why the change breaks sdcard at low speeds :)